plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/functions/tablechanges/TableChangesProcessorProvider.java

```diff
@@ -54,14 +55,14 @@ public TableChangesProcessorProvider(
     @Override
     public TableFunctionSplitProcessor getSplitProcessor(ConnectorSession session, ConnectorTableFunctionHandle handle, ConnectorSplit split)
     {
-        return new TableChangesFunctionProcessor(
+        return new ClassLoaderSafeTableFunctionSplitProcessor(new TableChangesFunctionProcessor(
```

Comment:
is it fixing a bug?
would you be able to describe the bug being fixed in end-user visible terms?

Reply:
Added to description

Reply:
thanks!
please update the Release notes section too

Reply:
I did update it, I think it is not user visible. Should I somehow change it ?

Reply:
discussed offline; this is a bug fix

please change the commit title from "Introduce ClassLoaderSafeTableFunctionSplitProcessor" to something that indicates it is a fix (perhaps "Fix ....")